examples/advanced/room_bot.py

```diff
@@ -0,0 +1,226 @@
+import asyncio
+import re
+import time
+from datetime import datetime
+from typing import List
+
+from apscheduler.schedulers.asyncio import AsyncIOScheduler
+
+from wechaty import Contact, Room, Wechaty, get_logger, Message
+
+welcome = """=============== Powered by Wechaty ===============
+-------- https://github.com/Chatie/wechaty --------
+Hello,
+I'm a Wechaty Botie with the following super powers:
+1. Find a room
+2. Add people to room
+3. Del people from room
+4. Change room topic
+5. Monitor room events
+6. etc...
+If you send a message of magic word 'ding',
+you will get a invitation to join my own room!
+__________________________________________________
+Hope you like it, and you are very welcome to
+upgrade me for more super powers!
+Please wait... I'm trying to login in..."""
+HELPER_CONTACT_NAME = '李卓桓'
+
+print(welcome)
+log = get_logger('RoomBot')
+
+
+async def checkRoomJoin(bot, room, inviteeList, inviter):
+    try:
+        user_self = bot.user_self()
+        if inviter.id != user_self.contact_id:
+            await room.say('RULE1: Invitation is limited to me, the owner only. '
+                           'Please do not invite people without notify me.' + inviter)
+            await room.say('Please contact me: by send "ding" to me, I will re-send you a invitation. '
+                           'Now I will remove you out, sorry.' + ''.join(inviteeList))
+            await room.topic('ding - warn ' + inviter.name())
+            scheduler = AsyncIOScheduler()
+            for i in inviteeList:
+                scheduler.add_job(room.delete, args=[i], seconds=10)
+            scheduler.start()
+        else:
+            await room.say('Welcome to my room! :)')
+            welcomeTopic = ', '.join(map(lambda c: c.name, inviteeList))
+            await room.topic('ding - welcome ' + welcomeTopic)
+    except Exception as e:
+        log.exception(e)
+
+
+async def manageDingRoom(bot):
```

Comment:
Refer to the [function and function](https://www.python.org/dev/peps/pep-0008/#function-and-variable-names) name style, we should keep the name of function and variable lowercase.